### ISSUE TITLE
Package.getStylesheetsPath is deprecated.

### ISSUE THREAD

**Issue by mrenfinger** [angular-ui/AngularJS-Atom#34]
Store package style sheets in the `styles/` directory instead of `stylesheets/` in the `angularjs` package
```
Package.getStylesheetsPath (/Users/michaelenfinger/Desktop/Atom.app/Contents/Resources/app.asar/src/package.js:464:9)
Package.getStylesheetPaths (/Users/michaelenfinger/Desktop/Atom.app/Contents/Resources/app.asar/src/package.js:475:32)
Package.loadStylesheets (/Users/michaelenfinger/Desktop/Atom.app/Contents/Resources/app.asar/src/package.js:457:38)
<unknown> (/Users/michaelenfinger/Desktop/Atom.app/Contents/Resources/app.asar/src/package.js:187:19)
```

**Comment by joshuajabbour** [comment]
Fixed in `0.2.0`. Not sure why automatic upgrade isn't working, but it can be upgraded manually with `apm install angularjs@0.2.0`.

**Comment by seronal** [comment]
I think this had to with this property in package.json:
```javascript
  "engines": {
    "atom": ">=0.174.0, <2.0.0"
  },
```

">=0.174.0, <2.0.0" does not seem to be a valid range, so the following was returning false
```javascript
semver.satisfies('0.200.0', '>=0.174.0, <2.0.0')
``` 

I just created a PR to fix this.

**Comment by outsideris** [comment]
I expect It fixed in #39 . I will publish new version.
